### DISCUSSION TITLE
Multiple containers using symfony-docker

### DISCUSSION THREAD

**Discussion by docmg** [dunglas/symfony-docker#395] [Q&A] [answered]
Is it possible to start multiple instances of the container with different domain names? I want to have two Symfony applications running on the same system. 

For example: 
- foo.example.com
- bar.example.com

As i understand, i need to setup a proxy server to forward requests to the containers depending on the domain. Does anyone know good solutions for this purpose?

I was trying to use NGINX Proxy Manager (NPM), but it's not working for me if the container i forward to is already using https.
Currently im thinking to configure Caddy to use only the port 80 and manage ssl-certificates in NPM, but im not sure if it's a right solution.

**Comment by maxhelias** [comment, marked as answer]
Caddy can be used as a proxy server.
And I'm not sure which case you want but you can configure multi domain on caddy :
 - that point to the same php container with the 2 applications.
 - with 2 php services with one application each (in that case, you need to configure `php_fastcgi` with the appropriate socket each).

**Reply by docmg** [reply]
Thank you for your response. I think the best approach for me is to manage the certificates using NPM and just serve Symfony over Port 80 using Caddy.

Is it possible to easily disable Port 443 and Certificate generation in this Template? I've been trying since yesterday and unfortunately i can't disable it (I don't have any experience with caddy atm).

**Reply by maxhelias** [reply]
See this option : https://caddyserver.com/docs/caddyfile/options#auto-https
